content/implementations/jvm.md

```diff
@@ -9,9 +9,9 @@ title: Cucumber JVM
 
 Cucumber-JVM is a Cucumber implementation for the most popular JVM languages.
 
-This document is the reference for features that are specific to Cucumber-JVM.
+This document is the reference for functionality specific to Cucumber-JVM.
 
-Please see the [General Reference](/gherkin/gherkin-reference/) for features common to all Cucumber implementations.
+Please see the [General Reference](/gherkin/gherkin-reference/) for functionality common to all Cucumber implementations.
```

Comment:
I really like using 'functionality' here, instead of "features" - which has a different meaning in this context :)